### ISSUE TITLE
How to connect from my PC server that run on the phone?

### ISSUE THREAD

**Issue by bladyle** [Genymobile/gnirehtet#450]
I can see that my phone is on different network but is there a way to connect to the same network so I can access a server runned on my phone?

**Comment by rom1v** [comment]
Not with gnirehtet. You can do the opposite (access a server running on your computer from your phone, using the address `10.0.2.2`).

**Comment by bladyle** [comment]
Yes already using the opposite and the app is very good for that - thanks for this! I was hopping that there is way that I don't know to get access to the server on the phone (put the tun0 adapter in the same network or something like that)
